### ISSUE TITLE
Ignore typography / punctuation (apostrophes, quotes, …)

### ISSUE THREAD

**Issue by ya-d** [h5p/h5p-blanks#83]
It would be great to have an option to ignore typography. Could be implanted as a pool of unicode characters which are treated as the same character.

A common use case are [apostrophes](https://apostrophe.how) were `don’t` and `don't` are treated as separate answers. Same for [straight and curly quotes](https://practicaltypography.com/straight-and-curly-quotes.html) and similar punctuation. There are several reports at the [feedback section](https://h5p.org/fill-in-the-blanks).

Related: #61. Thanks for considering!

**Comment by DegrangeM** [comment]
This is easy to add.

So `'` = `’` = `‘` and `"`=`“` = `”` ?

Should `'` = `"` ?

Also should `.` = `,` → this might be very nice for maths in country where decimal use `,`

Also should `'` = ` ` (empty) ?

**Comment by dodsoner** [comment]
I wanted to add that this issue is breaking key content, as in this example of fill in the blank content for English language learners (specifically, the answers that require _don't_ or _doesn't_). 

One big driver of this is that iPhones now insert the curly quotes by default, unless users have turned off those features. So, this typography bug affects a huge number of user devices.

**Comment by otacke** [comment]
A fix for this issue has been released just a couple of days after this issue was opened. It's in since version 1.12.11 of Fill in the Blanks. You might want to check what version you are running.

I just checked on iOS 14.7 and left single quotation marks and right single quotation marks that the iOS keyboard might emit are converted to apostrophes within blanks nicely. If it's not working on your end, I suggest that you provide more information - ideally on the [H5P forum](https://h5p.org/forum/9) which is the central hub for reporting issues.

**Comment by dodsoner** [comment]
Thank you, @otacke . Should this issue get closed out? 

I'm indeed running an older version, and have updated. I'm still trying to make sure I've cleared out the caches, but the error is still happening. I'll give it a few more tests over time and make sure everything is pulling from the 1.12.11 version, and will head to the forum with any other issues. 

And thanks for checking on your iOS device. I think what you wrote means that when you type something like _doesn't_, the iOS keyboard is now converting to the vertical tick, or &apos;. If so, that's great. Thank you.

**Comment by otacke** [comment]
@dodsoner Yes, I assume someone from the H5P core team could close this issue. And yup, that's what I technically meant: The left single quotation marks (`‘`) and right single quotation marks (`’`) (what you call curly quotes) are not used by iOS if a text input field has the attribute `spellcheck` set to `false`.

The code was added in https://github.com/h5p/h5p-blanks/commit/69ab18d878e57a86c74a9a76949ea5dca66164da#diff-872197d92b619c4a344ec34c1654df7874cb5c6bb7ad355bc42e81d721aa0535 and as you can see it has been around since v1.12.11. The latest releases version is v1.12.15 with 1.13.0 coming up next. Make sure to upgrade existing content if you've been running v1.11.x or earlier, otherwise you will only notice an effect for newly created content.

**Comment by otacke** [comment]
@dodsoner I wonder if this ticket could be closed by now?

**Comment by otacke** [comment]
@dodsoner Could you please close this ticket if you think it could?

**Comment by dodsoner** [comment]
@otacke I didn't open the ticket, nor do I have the ability to close it. I think your close ticket request is meant for @ya-d

**Comment by otacke** [comment]
@dodsoner Sorry, you are absolutely right, of course.
@ya-d Could you please close this ticket if you think it could?

**Comment by otacke** [comment]
@ya-d I'll assume that this ticket can be closed. If you disagree, please feel free to open a new one stating the problem.
